android-components/components/service/pocket/src/main/java/mozilla/components/service/pocket/PocketStoriesConfig.kt

```diff
@@ -47,8 +47,14 @@ class PocketStoriesConfig(
  * Configuration for sponsored stories request indicating parameters used to set spoc content.
  *
  * @property siteId ID of the site parameter.
+ * @property country Value of the country parameter.
+ * @property city Value of the city parameter.
```

Comment:
Could we add a bit more context for the future and say that this should be used with care as it allows to override the ip location and receive a different set of sponsored stories which may not be suited for the real location?